__tests__/components/editor/PropertyComponent.test.js

```diff
@@ -98,11 +99,32 @@ describe('<PropertyComponent />', () => {
         }
       }
 
-      const wrapper = shallow(<PropertyComponent propertyTemplate={template} />)
+      const wrapper = shallow(<PropertyComponent propertyTemplate={template}
+                               reduxPath={["http://id.loc.gov/ontologies/bibframe/note"]}/>)
       expect(wrapper.find('Connect(InputListLOC)').length).toEqual(0)
       expect(wrapper.find('Connect(InputLookupQA)').length).toEqual(0)
       expect(wrapper.find('Connect(InputLiteral)').length).toEqual(0)
     })
   })
 
+  it('error if <PropertyComponent /> is missing reduxPath props', () => {
+    const template = {
+      "propertyURI": "http://id.loc.gov/ontologies/bibframe/note",
+      "type": "resource",
+      "valueConstraint": {
+        "valueTemplateRefs": [
+          "resourceTemplate:bf2:Note"
+        ],
+        "useValuesFrom": []
+      }
+    }
+    console.error = jest.fn()
+    const wrapper = shallow(<PropertyComponent index={1}
```

Comment:
If we're only using the `wrapper` variable to appease eslint on line 125, what about removing the variable entirely? So, like:

```javascript
shallow(<PropertyComponent index={1} propertyTemplate={template} rtId={'resourceTemplate:test'} />)
expect(console.error).toHaveBeenCalledTimes(1)
```

Reply:
Good idea; fixed in commit.